client/components/external-link/docs/example.jsx

```diff
@@ -20,7 +20,7 @@ export default React.createClass( {
 					<a href="/devdocs/design/external-link">External Link</a>
 				</h2>
 				<Card>
-					<p><ExternalLink icon={ true } href="https://wordpress.org" >WordPress.org</ExternalLink></p>
+					<p><ExternalLink icon={ true } target="_blank" href="https://wordpress.org" >WordPress.org</ExternalLink></p>
```

Comment:
Ah, yes there is:

`/.../Sites/wp-calypso/client/components/external-link/docs/example.jsx
0:0  warning  File ignored because of a matching ignore pattern. Use "--no-ignore" to override`

I've added the eslint-disable comment now.